src/map.cpp

```diff
@@ -3496,7 +3526,8 @@ bool map::open_door( const tripoint &p, const bool inside, const bool check_only
         }
 
         if(!check_only) {
-            sounds::sound(p, 6, "", true, "open_door", furn.id);
+            sounds::sound(p, 6, "", true, "", "");
+            sfx::play_variant_sound( "open_door", ter.id, sfx::get_heard_volume(p));
```

Comment:
"Anyway, this PR has huge potential."

I think so too! Thanks so much for going through it and finding all those things. I will really really try to fix them.

Only, I don't really have my head around this coding stuff. I get what you are saying some of the time, and I will go through and try and compile with the changes you suggested, but I push pixels around!  I mean, I do it in Linux and I know _some_ coding, enough to bash this into compiling. But I don't know what you are talking some of the time.

I will try and be more specific when I run in to errors trying to make the changes you specified.